plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt

```diff
@@ -123,10 +124,12 @@ open class HtmlRenderer(
             node.dci.kind == ContentKind.Cover -> div("cover $additionalClasses") { //TODO this can be removed
                 childrenCallback()
             }
+            node.dci.kind == ContentKind.Deprecation -> div("deprecation-block") { childrenCallback() }
```

Comment:
`ContentKind.Deprecation` seems to be more universal and future-proof. Right now it's indeed a block, but maybe after UX redesign it'll be hint or a spoiler or something. Then renaming public API would be painful 

I thought about renaming the style, but having `div("deprecation")` seems strange to me, for some reason I want to add some other noun to it - `deprecation-info`, `deprecation-block`, `deprecation-content`, etc

I've changed the div style to `deprecation-details` so that it's also more universal, hopefully it's common ground :)